### PR TITLE
[WIP] 1.11.2

### PR DESCRIPTION
**Pull request by skmedix**
On the list is still many to do, Bukkit has many changes.

### CODE REVIEW

File: src/main/java/blue/lapis/pore/converter/type/material/MaterialConverter.java [hunk 1]
```diff
@@ -42,228 +42,243 @@ private MaterialConverter() {
 
     private static final TypeConverter<Material, BlockType> BLOCK_TYPE_CONVERTER =
             TypeConverter.builder(Material.class, BlockType.class)
-                    .add(Material.AIR, BlockTypes.AIR)
```
Comment (skmedix) [inline src/main/java/blue/lapis/pore/converter/type/material/MaterialConverter.java]: I know.. Tabs

**Comment by Maxqia** [comment]
Erm, I had the Material Converter sorted previously by how Bukkit did it for easier upgrading...

**Comment by jon100456** [comment]
can i have the compile jar? because i want to test it fully

**Comment by CyberSpongie** [comment]
https://hastebin.com/amuqukovaz.tex there are error you need to fix

**Comment by skmedix** [comment]
@jon100456
>fatal: Not a git repository (or any of the parent directories): .git
>Failed to apply patches to Porekit (from Porekit)`

Really? **[For sure?](https://travis-ci.org/skmedix/PoreRT/builds/194216593)**
There is no problems with compilation process,
but anyway server will crash because thats still [WIP].

**Comment by Maxqia** [comment]
Is this ready for merging?

**Comment by sgdc3** [comment]
@skmedix Any news?

**Comment by skmedix** [comment]
I'll try to finish this at the weekend.

**Comment by skmedix** [comment]
Closing due to lack of time. Sorry guys.

**Comment by sgdc3** [comment]
:(

**Pull request closed without merging**